product_docs/docs/postgres_distributed_for_kubernetes/1/quickstart.mdx

```diff
@@ -54,6 +54,12 @@ contains the definition of a PGD cluster with two data groups and a global
 witness node spread across three regions. Each data group consists of two data nodes
 and a local witness node.
 
+!!! NoteAvailability zones 
+    When creating Kubernetes clusters in different availability zones for cross-regional replication, 
+    ensure the clusters can communicate with each other by enabling network connectivity. To ensure 
+    pods and services are reachable by all Kubernetes clusters, deploy a network connectivity application like 
+    [Submariner](https://submariner.io/) on every cluster. 
+
```

Comment:
```suggestion
!!! NoteAvailability zones 
    When creating Kubernetes clusters in different availability zones for cross-regional replication, 
    ensure the clusters can communicate with each other by enabling network connectivity. Specifically, every service created with a `-node` or `-group` suffix must be discoverable by all other `-node` and `-group` services. This can be achieved by deploying a network connectivity application like 
    [Submariner](https://submariner.io/) on every cluster. 

```